webapp/src/components/AssetCard/AssetCard.container.ts

```diff
@@ -28,25 +25,20 @@ const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
   }
 
   const openRentalId = getOpenRentalId(asset)
-  let rentalPricePerDay: string | null = null
-  if (rental && rental.status === RentalStatus.OPEN) {
-    rentalPricePerDay = getMaxPriceOfPeriods(rental)
-  } else if (!rental && openRentalId) {
-    const rentalOfNFT = getRentalById(state, openRentalId)
-    if (rentalOfNFT && rentalOfNFT.status === RentalStatus.OPEN) {
-      rentalPricePerDay = getMaxPriceOfPeriods(rentalOfNFT)
-    }
-  }
+  let rentalOfNFT: RentalListing | null = openRentalId
+    ? getRentalById(state, openRentalId)
+    : null
 
   return {
-    rentalPricePerDay,
     price,
     showListedTag:
       Boolean(view === View.CURRENT_ACCOUNT && price) &&
       getLocation(state).pathname !== locations.root(),
     isClaimingBackLandTransactionPending: isNFT(asset)
       ? isClaimingBackLandTransactionPending(state, asset)
-      : false
+      : false,
+    rental: rentalOfNFT,
+    showRentalBubble: view === View.CURRENT_ACCOUNT
```

Comment:
```suggestion
    showRentalBubble: view === View.CURRENT_ACCOUNT &&
      getLocation(state).pathname !== locations.root(),
```
Should we add the `&&
      getLocation(state).pathname !== locations.root()` here as it does the `showListedTags`? I think it should have the same behavior. If I recall correctly, it was added to solve a bug where the tag was shown in the home page when coming back from the current account view.

Reply:
You're right, great catch, if I didn't add that, the chips / bubbles will be shown in the home page, changing it!